pkgs/development/tools/build-managers/dub/default.nix

```diff
@@ -46,7 +60,7 @@ stdenv.mkDerivation rec {
     rm test/issue990-download-optional-selected.sh
     rm test/issue877-auto-fetch-package-on-run.sh
     rm test/issue1037-better-dependency-messages.sh
-    rm test/issue1040-run-with-ver.sh
+    # rm test/issue1040-run-with-ver.sh
```

Comment:
```suggestion
```

Please remove all commented lines and -f from rm. Otherwise we won't notice if a file is removed.

Reply:
do I rebase (or make a branch) make changes (force push to self) and submit a modified patch to merge, also taking into account i077 comments below? ... Forgive the dumb question as it must have been asked hundreds of times before. Steps, should I: checkout master, create a branch say D-lang with the changes and commit message; how do i resubmit to this issue number. Or is this the wrong way to think about it? (sorry, nixpkgs dlang breakage finally got me to interact with github, not a bad thing perhaps but (not at all originally intended and) it is new and unplanned (the dub change mod is an old patch I thought to share in the circumstances)). Thanks, will return to later.

Reply:
Seems like you're opening a PR against the master branch of your fork. Like you said, you should make a new branch off of master, commit your changes, then open a new PR. You'll have to close this one since it uses the wrong source branch.